packages/@apphosting/adapter-angular/src/bin/build.ts

```diff
@@ -10,22 +10,28 @@ import {
 
 const root = process.cwd();
 
-// Determine root of project to build
+// Determine which build runner to use
+let cmd = process.env.MONOREPO_COMMAND || DEFAULT_COMMAND;
```

Comment:
Could we fetch all of the Monorepo related env vars all at once maybe in some function and have it all documented there? I'm having a bit of a hard time following all of the different monorepo related env vars.

Also it might be nice to have the Monorepo related code extracted out to a new file in general with their own set of unit tests too especially if this may get more complex as time goes on. Doesn't have to be in the pr, but might be good to add a TODO

Reply:
Yes this is absolutely something that should be done. Unfortunately there's a non-trivial amount of work to refactor this code into another module that can be imported by all adapters. In the interest of getting these fixes out in time for I/O I haven't addressed it in this PR, but it's at the top of my TODO list for sure